resources/views/join.blade.php

```diff
@@ -49,14 +62,60 @@
       </span>
         </p>
       </div>
+      <!-- END Email Field -->
+
+      <!-- Guardian First Name Field -->
+      <div class="field">
+        <label class="label">Parent / Guardian First Name</label>
+        <div class="control">
+          <input id="guardianFirst" class="input" type="text" placeholder="Parent or Guardian First Name" name="guardianFirst" onchange="hideErrors()">
+        </div>
+      </div>
+      <!-- END Guardian First Name Field -->
+
+      <!-- Guardian Last Name Field -->
+      <div class="field">
+        <label class="label">Parent / Guardian Last Name</label>
+        <div class="control">
+          <input id="guardianLast" class="input" type="text" placeholder="Parent or Guardian Last Name" name="guardianLast" onchange="hideErrors()">
+        </div>
+      </div>
+      <!-- END Guardian Last Name Field -->
+
+      <!-- Guardian Email Field -->
+      <div class="field">
+        <label class="label">Parent / Guardian Email</label>
+        <p class="control has-icons-left has-icons-right">
+          <input id="guardianEmail" class="input" type="email" placeholder="Parent or Guardian Email" name="guardianEmail" onchange="hideErrors()">
+          <span class="icon is-small is-left">
+        <i class="fa fa-envelope"></i>
+      </span>
+          <span class="icon is-small is-right">
+        <i class="fa fa-check"></i>
+      </span>
+        </p>
+      </div>
+      <!-- END Guardian Email Field -->
+
+      <!-- Guardian Phone Number -->
+      <div class="field">
+        <label class="label">Parent / Guardian Phone Number</label>
+        <div class="control">
```

Comment:
```suggestion
        <div class="control has-icons-left">
```

And then after the phone input:
`<span class="icon is-small is-lefft">
 	<i class="fa fa-phone" aria-hidden="true"></i>
 </span>`